lib/github-service.js

```diff
@@ -1,8 +1,16 @@
 const { Octokit } = require('@octokit/rest');
-const { buildMarkdownText } = require('./utils');
+const { buildMarkdownText, buildPullRequestComment } = require('./utils');
 
 const hasCollectionSucceed = (assertions) => assertions.failed_count === 0;
 
+const getAssociatedPullRequestNumberFromSha = async (octokit,repository, commitSha) => {
```

Comment:
async keyword should be removed since there is no await.

Reply:
Done